services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -301,12 +301,16 @@ export const DetailSection = ({
                     <Review className="preserve-spacing" heading="Change Made">
                       {adminChange.changeMade}
                     </Review>
-                    <Review
-                      className="preserve-spacing"
-                      heading="Change Reason"
-                    >
-                      {adminChange.changeReason}
-                    </Review>
+                    {!adminChange.changeMade.includes(
```

Comment:
why not use the absence of adminChange.changeReason to determine if this section should be shown? Im guessing there is a good reason?

Reply:
I thought about that and decided that No Change Reason on an entry that could have one is information someone might want.  I know that's kind of silly, but not like it was that hard to properly identify the enable/disable ones (even without a special type field :) )